src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -3107,5 +3103,467 @@ .locals init (char V_0)
   IL_006d:  ret
 }");
         }
+
+        [Fact]
+        public void New9PatternsSemanticModel_01()
+        {
+            // Tests for the semantic model in new patterns as of C# 9.0.
+            var source =
+@"
+using System;
+class Program
+{
+    void M(object o)
+    {
+        const int N = 12;
+        const char A = 'A';
+        const char Z = 'Z';
+        const char a = 'a';
+        const char z = 'z';
+        const char c0 = '0';
+        const char c9 = '9';
+        switch (o)
+        {
+            // Parenthesized patterns
+            case ((N, N)): break;
+            case (((long), (long))): break;
+            case ((N)): break;
+            // type patterns
+            case (int, int): break;
+            case (System.Int64, System.Int32): break;
+            case int: break;
+            // Conjunctive and disjunctive patterns
+            case (>= A and <= Z) or (>= a and <= z): break;
+            // Negated patterns
+            case not (> c0 and < c9): break;
+        }
+    }
+}
+";
+            var compilation = CreatePatternCompilation(source, options: TestOptions.DebugDll);
+            compilation.VerifyDiagnostics(
```

Comment:
>( [](start = 41, length = 1)

Consider removing the line break if expect no diagnostics. In other tests too.

Reply:
This is a convention used throughout this file, and I prefer it to emphasize that the diagnostics are empty, and to make it easy to change if that changes.

---
In reply to: [392351108](https://github.com/dotnet/roslyn/pull/42357#discussion_r392351108) [](ancestors = 392351108)